vpr/src/draw/draw_rr_edges.h

```diff
@@ -17,12 +17,53 @@
 
 #include "ezgl/graphics.hpp"
 
+/**
+ * @brief Draws the edge between two vertical channel nodes
+ */
 void draw_chany_to_chany_edge(RRNodeId from_node, RRNodeId to_node, short switch_type, ezgl::renderer* g);
+
+/**
+ * @brief Draws the edge between two horizontal channel nodes
+ */
 void draw_chanx_to_chanx_edge(RRNodeId from_node, RRNodeId to_node, short switch_type, ezgl::renderer* g);
+
+/**
+ * @brief Draws the edge between a horizontal channel node and a vertical channel node
+ * @param chanx_node The horizontal channel node
+ * @param chany_node The vertical channel node
+ * @param edge_dir The direction of the edge, FROM_X_TO_Y or FROM_Y_TO_X
+ * @param switch_type The type of switch used for the connection
+ * @param g The ezgl renderer 
+ */
 void draw_chanx_to_chany_edge(RRNodeId chanx_node, RRNodeId chany_node, enum e_edge_dir edge_dir, short switch_type, ezgl::renderer* g);
+
+/**
+ * @brief Draws the edge between an intra-cluster pin and a pin when flat routing is enabled. It does not matter whether prev_node is the intra-cluster pin or whether inode is the intra-cluster pin.
```

Comment:
"between an intra-cluster pin and a pin"
I think the second pin is an iner-cluster pin (or equivalently, you could call that a top-level cluster pin if you prefer).  I think the comment should make clear what the type of the second pin is.